### DISCUSSION TITLE
Continuous Archiving and Point-in-time Recovery

### DISCUSSION THREAD

**Discussion by giscus** [appscode/community#73] [KubeDB]
# Continuous Archiving and Point-in-time Recovery

KubeDB simplifies Provisioning, Upgrading, Scaling, Volume Expansion, Monitor, Backup, Restore for various Databases in Kubernetes on any Public & Private Cloud

https://kubedb.com/docs/v2024.11.18/guides/mariadb/pitr/overview/

**Comment by doteater** [comment]
```
data insert and switch wal
After each and every wal switch the wal files will be uploaded to backup storage
```
what is wal? Is this a typo or find/replace error?
